fabric8-docker-registry/src/main/fabric8/fabric8-docker-registry-storage-pvc.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fabric8-docker-registry-storage
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
Agreed - raised https://github.com/fabric8io/fabric8/issues/6694 to track the configureable / editable storage requests.